php/cubex/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "bajb/cubex-min",
+  "license": "MIT",
+  "type": "project",
+  "config": {
+    "optimize-autoloader": true,
+    "preferred-install": "dist",
+    "sort-packages": true
```

Comment:
Better to remove this (those steps are explicitly made on _bootstrap_ step)

https://github.com/the-benchmarker/web-frameworks/blob/1ee1a7b896f7c1c6ddb2cf6d694e0337b30e078a/php/Dockerfile#L28-L30